portal-api/api-tests/v1/validation/conversion-rate-date.api-test.js

```diff
@@ -0,0 +1,93 @@
+const checkConversionRate = require('../../../src/v1/validation/fields/currency-not-the-same-as-supply-contract-rules/conversion-rate-date');
+
+describe('validation - conversion rate date', () => {
+  const deal = {
+    details: {
+      submissionDate: null,
+    }
+  };
+
+  const errorList = {};
+
+  it('should return validation error if the conversion date year has a symbol in it', () => {
```

Comment:
Can we please add a test when month is `00` and year is `0000` please.

Reply:
done